core-services/prow/02_config/_plugins.yaml

```diff
@@ -951,7 +960,15 @@ config_updater:
         - ci
       gzip: true
       name: job-config-misc
-    ci-operator/jobs/openshift/kubernetes/*.yaml:
+    ci-operator/jobs/openshift-priv/kubernetes/openshift-priv-kubernetes-*.yaml:
```

Comment:
should be build01 too

Reply:
Hm, no other `ci-operator/jobs/...` one is?

Reply:
Do we still need jobs on api.ci?

Reply:
> Do we still need jobs on api.ci?

Looks like we do, as there are periodic jobs that use them. Which means `build01` will need them too as soon as we migrate one of those jobs there.